src/rules/core/id-references-for-certain-feeds-rule-spec.js

```diff
@@ -0,0 +1,157 @@
+const Model = require('../../classes/model');
+const ModelNode = require('../../classes/model-node');
+const OptionsHelper = require('../../helpers/options');
+const IdReferencesForCertainFeedsRule = require('./id-references-for-certain-feeds-rule');
+const ValidationErrorType = require('../../errors/validation-error-type');
+const ValidationErrorSeverity = require('../../errors/validation-error-severity');
+
+describe('IdReferencesForCertainFeedsRule', () => {
+  const rule = new IdReferencesForCertainFeedsRule();
+
+  describe('for kind FacilityUse/Slot or IndividualFacilityUse/Slot feeds', () => {
+    const model = new Model({
+      type: 'Slot',
+      validationMode: {
+        RPDEFeed: 'feed',
+        BookableRPDEFeed: 'feed',
+      },
+      rpdeKind: [
+        'FacilityUse/Slot',
+        'IndividualFacilityUse/Slot',
+      ],
+      imperativeConfigurationWithContext: {
```

Comment:
Given this same property is being used here for both RPDE kind and parent, suggest this should be renamed `imperativeConfigurationByRpdeKind` (and perhaps for clarity the existing property renamed to `imperativeConfigurationByParentPropertyName` or something more intuitive?)

This should make the configuration clearer